db/pages/accessibility.rb

```diff
@@ -94,3 +93,13 @@
   page.content = content
   page.save!
 end
+
+if SiteCustomization::Page.find_by(slug: "accessibility").nil?
+  page = SiteCustomization::Page.new(slug: "accessibility", status: "published")
+  generate_content(page)
+  I18n.available_locales.each do |locale|
+    I18n.locale = locale
+    translation = page.translations.build(locale: locale)
+    generate_content(translation)
+  end
```

Comment:
Maybe we could simplify the code a bit:

```suggestion
  I18n.available_locales.each do |locale|
    I18n.with_locale(locale) { generate_content(page) }
  end
```

What do you think? If you agree, we could do the same with the other pages :wink:.

Reply:
It looks much cleaner now. But I must admit I'm not quite sure how exactly it works 😮  . Applied it to the rest as this magic works 🥳

Reply:
The way Globalize works, when assigning `page.title`, it automatically assigns the title to a translation record based on `I18n.locale`. `I18n.locale` is different every time we canll `generate_content(page)` because it's inside the `I18n.with_locale(locale)` block.